src/client/common/configSettings.ts

```diff
@@ -98,6 +98,7 @@ export class JupyterSettings implements IWatchableJupyterSettings {
     public showVariableViewWhenDebugging: boolean = true;
     public newCellOnRunLast: boolean = true;
     public pylanceHandlesNotebooks: boolean = false;
+    public completionTriggerCharacters: string = '.%';
```

Comment:
We can leave this empty and let the vscode settings apply the defaults. This way defaults are in one place

Reply:
Sure good idea.